matching/score.go

```diff
@@ -26,6 +26,30 @@ func Score(places []Place) float64 {
 	return avgScore - avgDistance
 }
 
+func ScoreNoDistance(places []Place) float64 {
```

Comment:
Rename the function to make it clearer and it seems we don't need wrapper `singlePlaceScoreKnapSack`

Reply:
changed another name.